dev/bots/test.dart

```diff
@@ -344,7 +350,7 @@ Future<void> _buildRunnerTest(
   bool enableFlutterToolAsserts = false,
   bq.TabledataResourceApi tableData,
 }) async {
-  final List<String> args = <String>['run', 'build_runner', 'test', '--', '-rcompact', '-j1'];
```

Comment:
@jonahwilliams this _may_ have been the cause of the current failures, but I'lll try to figure that out for sure.

Reply:
Possible, but lets be a little conservative with these changes :)

Reply:
Agreed.  We've got enough noise right now, I don't want to be adding to it.